test/utils.jl

```diff
@@ -192,6 +192,10 @@ end
     end
   end
 
+  @testset "lecun_normal" begin
+    m = lecun_normal(100, 100)
+  end
```

Comment:
These are just taken from the doctests:
```suggestion
  @testset "lecun_normal" begin
    @test std(Flux.lecun_normal(10, 1000)) ≈ 0.032f0 rtol=0.01
    @test std(Flux.lecun_normal(1000, 10)) ≈ 0.317f0 rtol=0.01
  end
```
But it's also included in other tests above.